### DISCUSSION TITLE
[323220]Vagante {仅本体}

### DISCUSSION THREAD

**Discussion by kkaka-arch** [pjy612/SteamManifestCache#791] [Share]
### Steam 游戏ID

323220

### 密钥列表

```hash
323221;c303363ce4304b4fd265e8328cdacbebed351f897d5c3bbb151047fc645e7429
```

### 清单附件

[323221_7623889135158519514.zip](https://github.com/user-attachments/files/16912682/323221_7623889135158519514.zip)

**Comment by pjy612** [comment]
请确保附件上传完毕后 再提交

**Reply by kkaka-arch** [reply]
已重新上传
